### PR TITLE
feat: de version of schema update tutorial is added

### PR DESCRIPTION
**Pull request by Kate-Lyndegaard**

### CODE REVIEW

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+Laden sie zunächst die Quelldaten in hale»studio. Navigieren sie zum Menü Datei und wählen sie Importieren, anschließend wählen sie Quelldaten...
+
+Sobald die Quelldaten geladen sind, werden nacheinander möglicherweise viele Dialoge zu Zellentitäten angezeigt, bis alle Elemente aktualisiert wurden, um das neue Schema widerzuspiegeln. Für eine vollständige Aktualisierung muss der Endbenutzer für jede aktualisierte Referenz das von hale»studio gesuchte Element im Elementbaum manuell auswählen. In einigen Fällen identifiziert hale»studio das passende Element im neuen Schema korrekt (z. B. wenn sich nur der Namespace geändert hat, aber Typ und Eigenschaftsname gleich bleiben), in anderen Fällen ist eine direkte Benutzereingabe erforderlich. Es ist möglich, eine oder mehrere Zuordnungen im Dialog zu überspringen, wenn die Handhabung nicht ganz klar ist. Durch das Überspringen einer Zuordnung kann der Endbenutzer mit automatisch zugewiesenen oder erkannten Zuordnungen fortfahren, ohne den gesamten Vorgang abbrechen zu müssen. Nicht verarbeitete Zuordnungen können anschließend in der hale»studio-Benutzeroberfläche bearbeitet werden.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:55]:
```suggestion
Sobald die Quelldaten geladen sind, werden nacheinander möglicherweise viele Dialoge zu Zellentitäten angezeigt, bis alle Elemente aktualisiert wurden, um das neue Schema widerzuspiegeln. Für eine vollständige Aktualisierung müssen Nutzende für jede aktualisierte Referenz das von hale»studio gesuchte Element im Elementbaum manuell auswählen. In einigen Fällen identifiziert hale»studio das passende Element im neuen Schema korrekt (z. B. wenn sich nur der Namespace geändert hat, aber Typ und Eigenschaftsname gleich bleiben), in anderen Fällen ist eine direkte Benutzereingabe erforderlich. Es ist möglich, eine oder mehrere Zuordnungen im Dialog zu überspringen, wenn die Handhabung nicht ganz klar ist. Durch das Überspringen einer Zuordnung kann der Endbenutzer mit automatisch zugewiesenen oder erkannten Zuordnungen fortfahren, ohne den gesamten Vorgang abbrechen zu müssen. Nicht verarbeitete Zuordnungen können anschließend in der hale»studio-Benutzeroberfläche bearbeitet werden.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+<a className="image-news text-center col-xs-12 mt mb"><img src={require("/images/help/de/cell_entity_could_not_be_resolved.png").default} alt="" title="Cell entity matching" className="img-responsive center-block"/></a>
+
+**Schritt 4: Validierung prüfen und Transformationsprojekt speichern**
+
+Nachdem alle Cell entity matchings abgeschlossen sind, nehmen sie sich einen Moment Zeit, um die Validierung des aktualisierten Transformationsprojekts zu prüfen, um sicherzustellen, dass alle erforderlichen Schemaelemente ordnungsgemäß behandelt wurden. Öffnen sie in hale»studio das Fenster „Berichtsliste“, um den Status der Instanztransformation und der Instanzvalidierung anzuzeigen. Ein grünes Häkchen zeigt ein gültiges Projekt an. Eine gelbe Warnung zeigt an, dass das Projekt Zuordnungen mit Warnungen enthält, die im Eigenschaftenfenster überprüft werden können. Doppelklicken sie im Fenster „Berichtsliste“ auf die Instanztransformation und den Instanzvalidierungstext, um weitere Informationen zu Warnungen im Eigenschaftenfenster anzuzeigen.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:60]:
```suggestion
Nachdem alle Zellentitätsabgleiche abgeschlossen sind, nehmen sie sich einen Moment Zeit, um die Validierung des aktualisierten Transformationsprojekts zu prüfen und so sicherzustellen, dass alle erforderlichen Schemaelemente ordnungsgemäß behandelt wurden. Öffnen sie in hale»studio das Fenster „Report List“, um den Status der Instanztransformation und der Instanzvalidierung anzuzeigen. Ein grünes Häkchen zeigt ein gültiges Projekt an. Eine gelbe Warnung zeigt an, dass das Projekt Zuordnungen mit Warnungen enthält, die im Eigenschaftenfenster überprüft werden können. Doppelklicken sie im Fenster „Report List“ auf die Instanztransformation und den Instanzvalidierungstext, um weitere Informationen zu Warnungen im Eigenschaftenfenster anzuzeigen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+
+Speichern sie Ihr Projekt, sobald Ihre Prüfung abgeschlossen ist.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:62]:
```suggestion
Speichern sie ihr Projekt, sobald ihre Prüfung abgeschlossen ist.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+sie können das Transformationsprojekt INSPIRE_ProtectedSites direkt in Ihr hale»connect-Konto exportieren, sobald sie mit der Aktualisierung des Projekts fertig sind. Wenn sie Hilfe benötigen, besuchen sie das Hilfethema [Freigeben und Aktualisieren eines Transformationsprojekts auf hale»connect](http://help.halestudio.org/latest/index.jsp?topic=%2Feu.esdihumboldt.hale.doc.user%2Fhtml%2Ftasks%2Fhaleconnect%2Fshareproject.html) im Benutzerhandbuch von hale»studio für weitere Anweisungen.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:64]:
```suggestion
Sie können das Transformationsprojekt INSPIRE_ProtectedSites direkt in Ihr hale»connect-Konto exportieren, sobald sie mit der Aktualisierung des Projekts fertig sind. Wenn sie Hilfe benötigen, besuchen sie das Hilfethema [Freigeben und Aktualisieren eines Transformationsprojekts auf hale»connect](http://help.halestudio.org/latest/index.jsp?topic=%2Feu.esdihumboldt.hale.doc.user%2Fhtml%2Ftasks%2Fhaleconnect%2Fshareproject.html) im Benutzerhandbuch von hale»studio für weitere Anweisungen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+sie können ein Transformationsprojekt auch manuell hochladen.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:66]:
```suggestion
Sie können ein Transformationsprojekt auch manuell hochladen.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+
+### **Ein Online-Transformationsworkflows auf hale»connect aktualisieren**
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:68]:
```suggestion
### **Einen Online-Transformationsworkflow auf hale»connect aktualisieren**
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+
+**Schritt 1: Eine neue Schema erstellen**
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:70]:
```suggestion
**Schritt 1: Ein neues Schema erstellen**
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+
+Das aktualisierte Transformationsprojekt, mit dem sie arbeiten, transformiert deutsche Protected Sites-Daten in INSPIRE Protected Sites v5.0. Diese INSPIRE-Featuretypen befinden sich im Schema INSPIRE Protected Sites 5.0, daher erstellen sie dieses Schema auf hale»connect. hale»connect enthält eine große Anzahl von Schemavorgaben, darunter alle aktualisierten INSPIRE-Schemata.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:72]:
```suggestion
Das aktualisierte Transformationsprojekt, mit dem sie arbeiten, transformiert deutsche Schutzgebiet-Daten in INSPIRE Protected Sites v5.0. Diese INSPIRE-Featuretypen befinden sich im Schema INSPIRE Protected Sites 5.0, daher erstellen sie dieses Schema auf hale»connect. hale»connect enthält eine große Anzahl von Schema-Presets, darunter alle aktualisierten INSPIRE-Schemata.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+    * Wählen sie das INSPIRE Protected Sites 5.0-Schema-Preset aus dem Aufklappmenü verfügbarer Presets aus.<img src={require("/images/help/de/schema_5_de.png").default} alt="" title="Ein neues Schema von einem Preset anlegen" className="img-responsive img-inline-help"/>
+
+    * Informationen ergänzen. Geben sie den Namen der Autorin oder des Autors, den Typ und die Version ihres Schemas an. Wenn sie fertig sind, klicken sie auf &laquo;Weiter&raquo;.
+
+    * Anlegen bestätigen. Überprüfen sie in einem abschließenden Schritt ihr Schema. Sie haben die Option, Ihrem Schema ein Thema hinzuzufügen oder ein solches für ihr Schema zu erstellen. Wenn sie fertig sind, klicken sie auf &laquo;Anlegen&raquo; um ihr Schema zu speichern.
+
+
+**Schritt 2: Featuretypen des Ziel-Themas aktualisieren**
+
+Der nächste Schritt besteht darin, den Abschnitt „Featuretypen“ Ihres INSPIRE Protected Sites-Themas zu aktualisieren, um die Featuretypen aus dem INSPIRE Protected Sites 5.0-Schema zu verwenden.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:90]:
```suggestion
Der nächste Schritt besteht darin, den Abschnitt „Objektarten“ ihres INSPIRE Protected Sites-Themas zu aktualisieren, um die Objektarten aus dem INSPIRE Protected Sites 5.0-Schema zu verwenden.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+
+Navigieren sie zunächst zu Ihrem vorhandenen Protected Sites-Thema auf hale»connect. Wählen sie den Abschnitt „Featuretypen“ aus.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:92]:
```suggestion
Navigieren sie zunächst zu ihrem vorhandenen Protected Sites-Thema auf hale»connect. Wählen sie den Abschnitt „Objektarten“ aus.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+*	Schema definieren. Definieren sie als nächstes das Schema, das sie nutzen wollen. Die Schemadefinition hängt von der Erstellungsmethode ab, die sie im vorherigen Schritt ausgewählt haben.
+
+    * Wählen sie das INSPIRE Protected Sites 5.0-Schema-Preset aus dem Aufklappmenü verfügbarer Presets aus.<img src={require("/images/help/de/schema_5_de.png").default} alt="" title="Ein neues Schema von einem Preset anlegen" className="img-responsive img-inline-help"/>
+
+    * Informationen ergänzen. Geben sie den Namen der Autorin oder des Autors, den Typ und die Version ihres Schemas an. Wenn sie fertig sind, klicken sie auf &laquo;Weiter&raquo;.
+
+    * Anlegen bestätigen. Überprüfen sie in einem abschließenden Schritt ihr Schema. Sie haben die Option, Ihrem Schema ein Thema hinzuzufügen oder ein solches für ihr Schema zu erstellen. Wenn sie fertig sind, klicken sie auf &laquo;Anlegen&raquo; um ihr Schema zu speichern.
+
+
+**Schritt 2: Featuretypen des Ziel-Themas aktualisieren**
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:88]:
```suggestion
**Schritt 2: Objektarten des Ziel-Themas aktualisieren**
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+
+Der nächste Schritt besteht darin, den Abschnitt „Featuretypen“ Ihres INSPIRE Protected Sites-Themas zu aktualisieren, um die Featuretypen aus dem INSPIRE Protected Sites 5.0-Schema zu verwenden.
+
+Navigieren sie zunächst zu Ihrem vorhandenen Protected Sites-Thema auf hale»connect. Wählen sie den Abschnitt „Featuretypen“ aus.
+<img src={require("/images/help/de/update_feature_types_de.png").default} alt="" title="Featuretypen aktualisieren" className="img-responsive img-inline-help"/>
+
+Klicken sie auf das rote „x“, um die vorhandenen Featuretypen zu löschen, und klicken sie auf &laquo;+ Neuen Typ hinzufügen&raquo;. Navigieren sie im angezeigten Dialogfeld zum Schema INSPIRE Protected Sites 5.0 und wählen sie &laquo;Alle hinzufügen&raquo;. Ihr vorhandenes Thema ist jetzt aktualisiert und verwendet die Featuretypen aus dem Schema INSPIRE Protected Sites 5.0.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:95]:
```suggestion
Klicken sie auf das rote „x“, um die vorhandenen Objektarten zu löschen, und klicken sie auf &laquo;+ Neuen Typ hinzufügen&raquo;. Navigieren sie im angezeigten Dialogfeld zum Schema INSPIRE Protected Sites 5.0 und wählen sie &laquo;Alle hinzufügen&raquo;. Ihr vorhandenes Thema ist jetzt aktualisiert und verwendet die Objektarten aus dem Schema INSPIRE Protected Sites 5.0.
```

File: i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+
+**Schritt 3: Das aktualisierte Transformationsprojekt auslösen und vorhandene Ressourcen erneut veröffentlichen**
+
+Nachdem sie Ihr Transformationsprojekt aktualisiert und die Featuretypen in Ihrem vorhandenen Ziel-Thema aktualisiert haben, können sie die Online-Transformation erneut ausführen, um Ihre veröffentlichten Ressourcen auf hale»connect zu aktualisieren.
```
Comment (JohannaOtt) [inline i18n/de/docusaurus-plugin-content-docs/current/tutorials/2018-04-28-schema-update-tutorial.md:99]:
```suggestion
Nachdem sie ihr Transformationsprojekt aktualisiert und die Objektarten in ihrem vorhandenen Ziel-Thema aktualisiert haben, können sie die Online-Transformation erneut ausführen, um ihre veröffentlichten Ressourcen auf hale»connect zu aktualisieren.
```

**Comment by Kate-Lyndegaard** [comment]
@JohannaOtt Thank you for your review. I've added all of your feedback.

**Review by JohannaOtt — APPROVED** [review]
LGTM

**Pull request merged**